src/HashtagHandler.js

```diff
@@ -0,0 +1,60 @@
+var log  = require('npmlog');
+var RemoteRoom = require("matrix-appservice-bridge").RemoteRoom;
+var TwitterHandler = require('./TwitterHandler.js').TwitterHandler;
+
+/**
+ * HashtagHandler - Handler for hashtag room creation and messaging
+ * @class
+ * @extends {external:TwitterHandler}
+ *
+ * @param  {MatrixTwitter}   twitter
+ * @param  {matrix-appservice-bridge.Bridge}   bridge
+ */
+var HashtagHandler = function (bridge, twitter) {
+  TwitterHandler.call(this,bridge,"#","hashtag");
+  this.twitter = twitter;
+}
+
+HashtagHandler.prototype.onRoomCreated = function (alias,entry){
+  console.log(entry);
```

Comment:
A good tip here: If you're having to add debug logging (I assume this is given it is a `console.log`) then most likely it will also be a useful thing to log properly, so just go ahead and do it.